dojo/utils.py

```diff
@@ -663,8 +663,12 @@ def get_punchcard_data(objs, start_date, weeks, view='Finding'):
                                         .values('date') \
                                         .annotate(count=Count('id')) \
                                         .order_by('date')
+        else:
+            raise ValueError("view parameter only supports values 'Finding' or 'Endpoint'")
+
         # return empty stuff if no findings to be statted
-        if severities_by_day.count() <= 0:
+        # don't make an additional sql query, use len()
+        if len(severities_by_day) <= 0:
```

Comment:
severities_by_day isn't queried to count() is faster. len will result in loading all the findings into memory, creation of Finding objects etc which is slower.